src/actions/wallet/signTransaction.test.ts

```diff
@@ -358,12 +358,42 @@ describe('custom (cip42)', () => {
         feeCurrency: '0x765de816845861e75a25fca122bb6898b8b1282a',
         maxFeePerGas: parseGwei('20'),
         maxPriorityFeePerGas: parseGwei('2'),
-        type: 'cip42',
       }),
     ).toMatchInlineSnapshot(
```

Comment:
(Note-to-self) Would love to know how snapshots are generated and pasted here. I read about Snapshot testing here: https://jestjs.io/docs/snapshot-testing#inline-snapshots. 

It sounds like these are generated "automagically":

> First, you write a test, calling .toMatchInlineSnapshot() with no arguments

> The next time you run Jest, tree will be evaluated, and a snapshot will be written as an argument to toMatchInlineSnapshot

> By default, Jest handles the writing of snapshots into your source code. However, if you're using prettier in your project, Jest will detect this and delegate the work to prettier instead (including honoring your configuration).